### ISSUE TITLE
IDISP006 should not be reported when IDISP002 is reported

### ISSUE THREAD

**Issue by jnm2** [DotNetAnalyzers/IDisposableAnalyzers#105]
It makes it a pain to suppress, even though there is a single root cause:

```cs
#pragma warning disable IDISP002 // Dispose member.
#pragma warning disable IDISP006 // Implement IDisposable.
            private readonly IEnumerator<T> enumerator;
#pragma warning restore IDISP006
#pragma warning restore IDISP002
```

Could IDISP006 stop being reported here, and the "Implement IDisposable" fix be registered for IDISP002?

**Comment by kuzyaka** [comment]
+1

In my case IDISP002 / IDISP006 are reported for a single root cause too.
